packages/manager/modules/pci/src/projects/project/analytics-data-platform/analytics-data-platform.html

```diff
@@ -18,6 +18,8 @@
             data-steins="$ctrl.steins"
             data-customer-regions="$ctrl.customerRegions"
             data-product-regions="$ctrl.clustersRegions"
+            data-is-list-page="true"
+            data-product-name="{{:: 'analytics_data_platform_title' | translate}}"
```

Comment:
**nitpick:** Maybe we could have consistency:

In some places we have:
```html
  data-product-name="{{:: 'translation_key' | translate}}"
```

compared to:
```html
  data-product-name=":: 'translation_key' | translate"
```

Reply:
Fixed